src/libCps/CpsRunEvm.cpp

```diff
@@ -741,6 +741,95 @@ void CpsRunEvm::HandleApply(const evm::EvmResult& result,
   }
 }
 
+bool CpsRunEvm::ProbeERC165Interface(CpsAccountStoreInterface& accStore,
```

Comment:
You probably want to split this out into a ProbeERC165Interface(ifaceId), and a call to ProbeERC165Interface(keccak256("scilla_handle_event", .. )) so that future checks for interface support don't have to replicate the generic ERC165 support code, but we can do that later..